### ISSUE TITLE
Add error message to Input components with validation errors

### ISSUE THREAD

**Issue by julianguyen** [ifmeorg/ifme#1085]
# Description 

Currently, in for our forms that use React components (Input, Form), we only display validation errors with an exclamation mark icon. That is not the best experience and it would be better if we also had accompanying error text. For folks who use screen readers, this is a particularly bad experience.

Let's add that! Please put the error message in i18n and use react-intl from the client to pull out that text.

# Screenshots

## Currently
![image](https://user-images.githubusercontent.com/3010728/46429188-afe5d980-c6fa-11e8-8d96-4957a8b2674f.png)

## Desired
![image](https://user-images.githubusercontent.com/3010728/46429110-7b721d80-c6fa-11e8-8446-280c569693dc.png)

# Do you want to be the assignee to work on this?

🚫

**Comment by Janice0218** [comment]
Can i take over it?

**Comment by DamonClark** [comment]
Yea go ahead! I can find something else to work on tomorrow :)

**Comment by julianguyen** [comment]
@Janice0218 Thanks for expressing your interest. Please check out https://github.com/ifmeorg/ifme/wiki/Join-Our-Slack for next steps. After that, assign yourself to the issue.

**Comment by Mera-Gangapersaud** [comment]
Hey, @Janice0218  are you still working on this?

**Comment by julianguyen** [comment]
Technically it's unassigned and our [rule](https://github.com/ifmeorg/ifme/wiki/Picking-Up-Issues) is that you have to assign yourself to the issue to be working on it @Mera-Gangapersaud

**Comment by conkytom** [comment]
Hey @Mera-Gangapersaud, I don't mean to step on any toes, but I gave this a shot.  Found an error message already in the i18n files.  I could only get it added to the top of the field next to the exclamation, I couldn't figure out how to add it underneath. https://github.com/ifmeorg/ifme/pull/1250

**Comment by Mera-Gangapersaud** [comment]
I looked at where the error message is added in the InputLabel component and its contained in a separate div from the text box. I'm not too sure, but I think the text box is the InputTextarea component. How would I go about adding a label below that without having to change the dynamic form?
